### PR TITLE
Review/updates for Astro docs

### PR DESCRIPTION
**Pull request by alexisintech**
Full editorial of Astro docs that were released today.

We're going to use external links for links that take you from the Astro SDK to another SDK (such as our Nextjs SDK). This is a temporary solution until we figure out how to deal with related/duped docs.

### CODE REVIEW

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 1]
```diff
@@ -38,32 +32,30 @@ Uninstall the community SDK and install the new offical SDK for Astro.
...
 
-**Bundled** \
-We have completely removed the option for an integration that uses the bundled clerk-js variant
-
-### Prefix for environment variable
+### Prefix for environment variables
 
-We changed the expected required prefix for environment variables available in client-side code. This was to align with [best practises](https://docs.astro.build/en/guides/environment-variables/) by Astro.
-This change is applicable to all [environment variables](https://clerk.com/docs/deployments/clerk-environment-variables) Clerk supports.
+The prefix for environment variables used in client-side code has been updated to align with Astro's [best practises](https://docs.astro.build/en/guides/environment-variables/). This change affects all [environment variables](https://clerk.com/docs/deployments/clerk-environment-variables) supported by Clerk.
```
Comment (jescalan) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:58]:
```suggestion
The prefix for environment variables used in client-side code has been updated to align with Astro's [best practices](https://docs.astro.build/en/guides/environment-variables/). This change affects all [environment variables](https://clerk.com/docs/deployments/clerk-environment-variables) supported by Clerk.
```

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 2]
```diff
@@ -94,9 +88,9 @@ This change is applicable to all [environment variables](https://clerk.com/docs/
 
 ```
 
-### Changes related to stores
+### Stores
 
-Submodule `/stores` was replaced with `/client`. For the full list of the stores see [here](docs/references/astro/overview#client-side-helpers)
+Submodule `/stores` was replaced with `/client`. The following changes are required to import the [stores](docs/references/astro/overview#client-side-helpers):
```
Comment (jescalan) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:93]:
```suggestion
Submodule `/stores` was replaced with `/client`. The following changes are required to import the [stores](/docs/references/astro/overview#client-side-helpers):
```

**Review by jescalan — APPROVED** [review]

File: docs/references/astro/locals.mdx [hunk 3]
```diff
@@ -48,14 +47,11 @@ if (Astro.request.method === "POST") {
...
 ```
+</CodeBlockTabs>
 
 ## Accessing the current user
 
-Current user data is important for data enrichment. You can use the `currentUser()` local to fetch the current user's data in your pages.
-
-Under the hood, this local:
-- uses the `GET /v1/users/me` endpoint.
-- counts towards the [Backend API Request Rate Limit](/docs/backend-requests/resources/rate-limits#rate-limits).
```
Comment (royanger) [inline docs/references/astro/locals.mdx:58]: We should be keeping some warning about this local counting towards rate limits. We do that, I think, everywhere else that the Next version of `currentUser()` is documented.

**Review by royanger — APPROVED** [review]

File: docs/references/astro/organization-store.mdx [hunk 4]
```diff
@@ -40,7 +40,7 @@ export default function Home() {
 
 The following example demonstrates how to implement pagination for organization memberships. The `memberships` state will be populated with the first page of the organization's memberships. When the "Previous page" or "Next page" button is clicked, the `fetchMemberships` function will be called to fetch the previous or next page of memberships.
 
-You can implement this pattern to any Clerk function that returns a [`ClerkPaginatedResponse`](/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response) object.
+You can implement this pattern to any Clerk function that returns a [`ClerkPaginatedResponse`](https://clerk.com/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response) object.
```
Comment (panteliselef) [inline docs/references/astro/organization-store.mdx:43]: ❓ What's the reason behind this change ? In order for me to know when i should use absolute vs relative

File: docs/references/astro/endpoints.mdx [hunk 5]
```diff
@@ -1,16 +1,14 @@
 ---
-title: Protect your Astro endpoints
-description: Learn how to use Clerk in Astro to protect your endpoints.
+description: Learn how to use Clerk's Astro SDK in your Astro endpoints.
```
[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/references/astro/endpoints.mdx:2]: it's essentially the same page as https://clerk.com/docs/references/nextjs/route-handlers
if someone is searching for something on the page, the headings of the sections are more clear and the search can lead them to the correct section.
is there a title you had in mind?

File: docs/references/astro/organization-store.mdx (hunk 4 above)
Comment (alexisintech) [reply docs/references/astro/organization-store.mdx:43]: we discussed it in the groupchat, and it's also written in the description of the pr: "We're going to use external links for links that take you from the Astro SDK to another SDK (such as our Nextjs SDK). This is a temporary solution until we figure out how to deal with related/duped docs."

File: docs/references/astro/locals.mdx (hunk 3 above)
Comment (alexisintech) [reply docs/references/astro/locals.mdx:58]: updated!

File: docs/references/astro/endpoints.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (royanger) [reply docs/references/astro/endpoints.mdx:2]: > it's essentially the same page as https://clerk.com/docs/references/nextjs/route-handlers if someone is searching for something on the page, the headings of the sections are more clear and the search can lead them to the correct section. is there a title you had in mind?

I don't love it, only as its not clear that the page name and the titles are related to someone new to using Clerk docs. There is no tree or anything to indicate that, so someone might not realize they are all related. That said, maybe I'm making too much out of this. 

Custom_Flows__Build_a_custom_email_or_SMS_OTP_authentication_flow

**Review by royanger — APPROVED** [review]
I replied to one thing, but overall this is good to go.
